index.ts

```diff
@@ -1 +1 @@
-export { YourWebComponent } from './src/your-webcomponent';
+export { UnderlinedTabBar as YourWebComponent } from './src/iaux-underlined-tab-bar';
```

Comment:
Remove the `as YourWebComponent`. I think this was an artifact from when it got renamed.